src/ucp/core/ucp_ep.inl

```diff
@@ -14,6 +14,7 @@
 #include "ucp_context.h"
 
 #include <ucp/wireup/wireup.h>
+#include <ucp/wireup/wireup_cm.h>
```

Comment:
no need, can be removed

Reply:
I'll clean up when CI finishes to be sure that there are no other failures